requirements.txt

```diff
@@ -127,6 +215,10 @@ rich==13.7.0 \
     --hash=sha256:5cb5123b5cf9ee70584244246816e9114227e0b98ad9176eede6ad54bf5403fa \
     --hash=sha256:6da14c108c4866ee9520bbffa71f6fe3962e193b7da68720583850cd4548e235
     # via hatch
+secretstorage==3.3.3 \
+    --hash=sha256:2403533ef369eca6d2ba81718576c5e0f564d5cca1b58f73a8b23e7d4eeebd77 \
+    --hash=sha256:f356e6628222568e3af06f2eba8df495efa13b3b63081dafd4f7d9a7b7bc9f99
+    # via keyring
```

Comment:
This is a good example of some of the issues with lock files and different platforms. I ended up having to regenerate my lock file in Linux because `keyring` requires `secretstorage` on the linux platform: 

https://github.com/jaraco/keyring/blob/f90ac2a712fa6d9b070d16a3eca6a4133cfdb422/setup.cfg#L21-L27